Manifest.toml

```diff
@@ -0,0 +1,122 @@
+[[Base64]]
```

Comment:
Are you sure the Manifest and the Project files are necessary? I've not seen them in any other packages on 1.0; I think attobot is supposed to create them when they become compulsory?

Reply:
I'm not quite sure -- I have not been able to make travis work without it.

Reply:
well then.